srtcore/buffer_rcv.h

```diff
@@ -142,7 +142,7 @@ class CRcvBuffer
     /// Query how many data has been continuously received (for reading) and available for reading out
     /// regardless of the TSBPD.
     /// TODO: Rename to countAvailablePackets().
-    /// @return size of valid (continous) data for reading.
+    /// @return size of valid (continuos) data for reading.
```

Comment:
This isn't correct either, but I think the word meant here is "contiguous". @maxsharabayko ?

Reply:
I meant "continuous" here, but missed one "i". :)